blueprints/automation/panhans/advanced_heating_control.yaml

```diff
@@ -331,6 +333,8 @@ blueprint:
               filter:
                 - domain:
                     - input_boolean
+                    - binary_sensor
+                    - timer
```

Comment:
Timers won't work here. Since they are not binary. To make them work there are some changes in the variable and trigger section needed.